### PR TITLE
feat: window playground v3

### PR DESCRIPTION
**Pull request by irainia**
This PR is to add window playground v3. Version 3 is the main version of playground. Version 1 and version 2 are moved to another package. In order to activate the version 1 and version 2 mode, the user needs to use flag `--v1-v2` when executing playground. Example:

```zsh
# execute window playground v3
optimus playground window

# execute window playground v1 and v2
opitmus playground window --v1-v2
```

### CODE REVIEW

File: docs/docs/concepts/intervals-and-windows.md [hunk 1]
```diff
@@ -123,10 +123,14 @@ dependencies: []
 
 Notice the window configuration is specified under field `task.window`. **Important** note, the `version` field decides which
 version of window capability to be used. Currently available window version are `version: 1`, `version: 2` and `version: 3`. Version 3 is recommended
-to be used as version 1 will soon be deprecated. To know the difference between the two version, run the `playground` feature for window:
+to be used as version 1 will soon be deprecated. To play around with window configuration, try `playground` feature for window:
```
Comment (arinda-arif) [inline docs/docs/concepts/intervals-and-windows.md:126]: `version 1 and 2 will soon be deprecated`

File: client/cmd/playground/window/model.go [hunk 2]
```diff
@@ -50,35 +72,39 @@ func (m *model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
...
 }
 
 func (m *model) View() string {
+	buff := new(bytes.Buffer)
+	table := tablewriter.NewWriter(buff)
+	table.SetAutoWrapText(false)
+	table.SetBorder(false)
+	table.SetRowLine(false)
+	table.SetColumnSeparator("")
+	table.Append([]string{m.generateWindowInputView(), m.generateWindowInputHintView()})
```
Comment (arinda-arif) [inline client/cmd/playground/window/model.go:100]: not crucial, but seems input tables location for V3 is slightly shifted (input tables are not in the same column with the result column)

File: client/cmd/playground/window/model.go [hunk 3]
```diff
@@ -89,130 +115,229 @@ func (m *model) View() string {
...
+press (shift+up) or (shift+w) to increment value
+press (shift+down) or (shift+s) to decrement value
 `
-	case pointToSize:
-		hint = `valid formats are:
-- nMmh: example 1M2h meaning 1 month 2 hours
-- mh: example 2h meaning 2 hours
+	case pointToTruncateToUnit:
+		hint = `valid values are:
+- None or <empty>: truncation happens based on the unit if size
```
Comment (arinda-arif) [inline client/cmd/playground/window/model.go:170]: `if` is typo?

File: client/cmd/playground/window/model.go [hunk 4]
```diff
@@ -89,130 +115,229 @@ func (m *model) View() string {
...
+		m.generateTruncateToView(),
+	})
+	table.Append([]string{
+		"location",
+		m.generateLocationView(),
 	})
 	table.Append([]string{
 		"job schedule",
-		m.generateSechduledTimeView(),
+		m.generateSechduleTimeView(),
```
Comment (arinda-arif) [inline client/cmd/playground/window/model.go:241]: typo `Sechdule`

File: docs/docs/concepts/intervals-and-windows.md (hunk 1 above)
Comment (irainia) [reply docs/docs/concepts/intervals-and-windows.md]: updated as suggested, with a little modification

File: client/cmd/playground/window/model.go (hunk 2 above)
Comment (irainia) [reply client/cmd/playground/window/model.go]: updated as suggested, but required a little refactor for more code readability

Comment (irainia) [reply client/cmd/playground/window/model.go]: updated as suggested, but required a little refactor for more code readability

File: client/cmd/playground/window/model.go (hunk 3 above)
Comment (irainia) [reply client/cmd/playground/window/model.go]: updated, with small update for better readability

File: client/cmd/playground/window/model.go (hunk 4 above)
Comment (irainia) [reply client/cmd/playground/window/model.go]: updated

**Review by arinda-arif — APPROVED** [review]

**Pull request merged**